docs/tutorials/vagrant.rst

```diff
@@ -767,15 +763,35 @@ Power On
     curl -k -X POST \
         -H 'Content-Type: application/json' -H 'Authorization: JWT <token>' \
         -d '{"name": "Graph.PowerOn.Node"}' \
-        https://localhost:9093/api/2.0/nodes/5799151faa0559c007dab5e3/workflows
+        https://localhost:9093/api/2.0/nodes/<nodeId>/workflows
 
 Install OS
 ^^^^^^^^^^^
+Three steps to install os.
+
+- step 1: OBM setting
+.. code-block:: REST
+    curl localhost:9090/api/current/nodes/<nodeId>/catalogs/bmc | jq '.' | grep "IP Address" 
```

Comment:
is this specific to Infrasim vnode, as I know, some vendors catalog's 'IP Address' don't have BMC IP